<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="AVK" failures="0" tests="1" name="practice.RetryAnalysePractice" time="0.026" errors="1" timestamp="2024-01-04T14:25:17 IST" skipped="0">
  <testcase classname="practice.RetryAnalysePractice" name="analyser" time="0.026">
    <error message="class PropertyFileUtility.RetryAnalyserImplementation cannot be cast to class org.testng.IRetryAnalyzer (PropertyFileUtility.RetryAnalyserImplementation and org.testng.IRetryAnalyzer are in unnamed module of loader &#039;app&#039;)" type="java.lang.ClassCastException">
      <![CDATA[java.lang.ClassCastException: class PropertyFileUtility.RetryAnalyserImplementation cannot be cast to class org.testng.IRetryAnalyzer (PropertyFileUtility.RetryAnalyserImplementation and org.testng.IRetryAnalyzer are in unnamed module of loader 'app')
at org.testng.internal.BaseTestMethod.computeRetryAnalyzerInstanceToUse(BaseTestMethod.java:843)
at org.testng.internal.BaseTestMethod.getRetryAnalyzerConsideringMethodParameters(BaseTestMethod.java:817)
at org.testng.internal.BaseTestMethod.getRetryAnalyzer(BaseTestMethod.java:681)
at org.testng.internal.invokers.TestInvoker.shouldRetryTestMethod(TestInvoker.java:549)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:726)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:227)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:957)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:200)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:848)
at org.testng.TestRunner.run(TestRunner.java:621)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:443)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:437)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:397)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </error>
  </testcase> <!-- analyser -->
</testsuite> <!-- practice.RetryAnalysePractice -->
